### PR TITLE
[Mobile Cards] Update Push cards tutorials

### PR DESCRIPTION
**Pull request by sandeep-tds**
Fixing the Push Tutorials after the single instance and multi-instance feature was released.

### CODE REVIEW

File: tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md [hunk 1]
```diff
@@ -12,394 +12,149 @@ author_profile: https://github.com/sandeep-tds
...
-- You have configured a connection between your SAP Business Application Studio and the SAP Mobile Services cockpit.
 
 ## Details
 ### You will learn
-  - Create a Web Page Matching card using a template in Business Application Studio
-  - Design and develop SAP Mobile Cards in Business Application Studio
-  - Deploy and Publish a card to SAP Mobile Services from Business Application Studio
-
-Add additional information: Background information, longer prerequisites
+  - how to modify the card type to single instance in Business Application Studio
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md:18]: Start with upper case.

File: tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md [hunk 2]
```diff
@@ -12,394 +12,149 @@ author_profile: https://github.com/sandeep-tds
...
+  - how to add subscription parameters to SAP Mobile cards
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md:19]: Start with upper case.

File: tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md [hunk 3]
```diff
@@ -12,394 +12,149 @@ author_profile: https://github.com/sandeep-tds
...
+  - how to deploy and publish a card to SAP Mobile Services from Business Application Studio
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-cards-create-sales-push/cp-mobile-cards-create-sales-push.md:20]: Start with upper case.

File: tutorials/cp-mobile-cards-push-sales-card/cp-mobile-cards-push-sales-card.md [hunk 4]
```diff
@@ -10,48 +10,48 @@ author_profile: https://github.com/sandeep-tds
...
-  - You are working with Postman client, CURL or any other API development platform.
+  - You have [created a push enabled card](cp-mobile-cards-create-sales-push).
+  - Postman client, CURL or any other API development platform
 
 ## Details
 ### You will learn
-  - Push a card to your Mobile Device from an external system
-  - View and delete a Web Page Matching Card
-  - Perform an action using push notification
+  - how to push a card to your mobile device from an external system
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-cards-push-sales-card/cp-mobile-cards-push-sales-card.md:18]: Start with upper case.

File: tutorials/cp-mobile-cards-push-sales-card/cp-mobile-cards-push-sales-card.md [hunk 5]
```diff
@@ -10,48 +10,48 @@ author_profile: https://github.com/sandeep-tds
...
+  - how to view and delete subscription cards on your mobile device
```
Comment (MichaelCzcz) [inline tutorials/cp-mobile-cards-push-sales-card/cp-mobile-cards-push-sales-card.md:19]: Start with upper case.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Hello @sandeep-tds, Thank you for your PR. Please correct those typos and I will merge your changes.

**Comment by thecodester** [comment]
@sandeep-tds Also, when this is merged, let me know which groups/missions to label free tier

**Pull request merged**